### PR TITLE
localstore/leveldb: using fatal instead of return error when the error should never happend

### PR DESCRIPTION
**Pull request by ngaut**

### CODE REVIEW

**Comment by shenli** [comment]
LGTM

**Comment by hanfei1991** [comment]
LGTM

File: store/localstore/goleveldb/goleveldb.go [hunk 1]
```diff
@@ -14,6 +14,7 @@
 package goleveldb
 
 import (
+	"github.com/ngaut/log"
```
Comment (hanfei1991) [inline store/localstore/goleveldb/goleveldb.go]: format code

**Comment by coocood** [comment]
```
batch := b.(*leveldb.Batch)
```
Will do.

**Pull request closed without merging**